tools/src/test/java/org/apache/kafka/tools/ConsumerPerformanceTest.java

```diff
@@ -136,7 +174,9 @@ public void testConfigWithoutTopicAndInclude() {
 
     @Test
     public void testClientIdOverride() throws IOException {
-        File tempFile = Files.createFile(tempDir.resolve("test_consumer_config.conf")).toFile();
+        Path configPath = tempDir.resolve("test_consumer_config.conf");
+        Files.deleteIfExists(configPath);
```

Comment:
This is not required, tempDir will be cleanup automatically after junit test finished.
Same thing in other places.